Source/DataLayer/EfClasses/Book.cs

```diff
@@ -101,20 +102,25 @@ public class Book
 			Category = category;
 
             // simple assigns
-            Title = title.Trim() ?? "";
-      Subtitle = subtitle?.Trim() ?? "";
-            Description = description?.Trim() ?? "";
+            UpdateTitle(title, subtitle);
+			Description = description?.Trim() ?? "";
             LengthInMinutes = lengthInMinutes;
             ContentType = contentType;
 
             // assigns with biz logic
             ReplaceAuthors(authors);
             ReplaceNarrators(narrators);
 		}
+        public void UpdateTitle(string  title, string subtitle)
```

Comment:
Thanks to which ever of you did this. I like that updating title and subtitle: A: has to be done together and B: is really intentional. Title and Subtitle should typically not change. Using this rather than casual property assignments makes this more clear.